Week 04/Lecture 07/Assignment2/assignment2/src/main/java/com/lecture7/assignment2/service/EmployeeServiceConstructor.java

```diff
@@ -0,0 +1,30 @@
+package com.lecture7.assignment2.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.lecture7.assignment2.entity.Email;
+import com.lecture7.assignment2.entity.Employee;
+
+@Service
+public class EmployeeServiceConstructor {
+
+    final EmailService emailService;
+
+    @Autowired
```

Comment:
Yes, I can remove the @Autowired annotation. This is because the class only have one constructor so Spring will applies implicit constructor injection by default.